jetty-hazelcast/src/main/config/modules/session-store-hazelcast-remote.mod

```diff
@@ -34,6 +34,6 @@ jetty.session.hazelcast.mapName=jetty-distributed-session-map
 jetty.session.hazelcast.hazelcastInstanceName=JETTY_DISTRIBUTED_SESSION_INSTANCE
 jetty.session.hazelcast.onlyClient=true
 jetty.session.hazelcast.scavengeZombies=false
-#jetty.session.hazelcast.configurationLocation=
```

Comment:
it must be still possible to configure hazelcast so I would prefer leave it but it's not in the xml and it should be maybe you can add it as well?